### PR TITLE
Clean up the routine of server

### PR DESCRIPTION
**Pull request by takahashi-tsc**
We use WaitGroup and context.

for #32

### CODE REVIEW

**Comment by takahashi-tsc** [comment]
Sorry but #32 and #40 should be considered together. I'll fix this commit.

**Comment by takahashi-tsc** [comment]
Completed.

File: cmd/server/amqp.go [hunk 1]
```diff
@@ -82,26 +83,27 @@ func runSubscriber(config *Config) {
...
+			break EVENTLOOP
+		case d, ok := <-msgs:
+			if ok {
+				dataText := strings.SplitN(string(d.Body), "/", 2)
 
-	go func() {
-		for d := range msgs {
-			dataText := strings.SplitN(string(d.Body), "/", 2)
+				dst, err := os.Create(confDirPath + dataText[0])
+				failOnError(err, "file create NG")
```
Comment (s1061123) [inline cmd/server/amqp.go]: Can we make it capitalize?

Comment (takahashi-tsc) [reply cmd/server/amqp.go]: This is an error message. In my understanding, error messages should begin with lower case.
Please discuss at the meeting. If my understanding is wrong, I will make another PR.

**Pull request merged**